tests/snmp/test_snmp_interfaces.py

```diff
@@ -159,14 +173,32 @@ def test_snmp_mgmt_interface(localhost, creds_all_duts, duthosts, enum_rand_one_
     for name in config_facts.get('MGMT_INTERFACE', {}):
         assert name in snmp_ifnames, "Management Interface not found in SNMP facts."
 
-def test_snmp_interfaces_mibs(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_all_duts):
+    # TODO: Remove this check after operational status of mgmt interface
+    # is implemented for multi-asic platform
+    if duthost.num_asics() == 1:
+        ports_list = []
+        _ = [ports_list.extend(config_facts.get(i, {}).keys())
+             for i in ['MGMT_INTERFACE']]
+        dut_facts = collect_all_facts(duthost, ports_list)
+        ports_snmps = verify_port_snmp(dut_facts, snmp_facts)
+        speed_snmp = verify_snmp_speed(dut_facts, snmp_facts, ports_snmps)
+        result = verify_port_ifindex(snmp_facts, speed_snmp)
+        pytest_assert(not result, "Unexpected comparsion of SNMP: {}".format(result))
+
+def test_snmp_interfaces_mibs(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_all_duts, enum_asic_index):
     """Verify correct behaviour of port MIBs ifIndex, ifMtu, ifSpeed,
        ifAdminStatus, ifOperStatus, ifAlias, ifHighSpeed, ifType """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    namespace = duthost.get_namespace_from_asic_id(enum_asic_index)
     hostip = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars['ansible_host']
     snmp_facts = localhost.snmp_facts(host=hostip, version="v2c", community=creds_all_duts[duthost]["snmp_rocommunity"])['ansible_facts']
+    config_facts = duthost.config_facts(host=duthost.hostname, source="persistent", namespace=namespace)['ansible_facts']
+
+    ports_list = []
+    _ = [ports_list.extend(config_facts.get(i, {}).keys())
```

Comment:
>_ =  [](http://example.com/codeflow?start=4&length=4)

This assignment looks weird. Could you explain why it is needed?

Reply:
This is the same logic as in the current test implementation:
```
    ports_list = []
    _ = [ports_list.extend(config_facts.get(i, {}).keys())
         for i in ['port_name_to_alias_map', 'PORTCHANNEL', 'MGMT_INTERFACE']]
 ```
This will be same as:
```
    for i in ['port_name_to_alias_map', 'PORTCHANNEL']:
        ports_list.extend(config_facts.get(i, {}).keys())
```

Reply:
Seems the below code is readable. Could you use it?

Reply:
Updated to make it clear.